app/backend/src/clj/teet/meta/meta_model.clj

```diff
@@ -19,6 +19,11 @@
   {:meta/modifier    (user-model/user-ref user)
    :meta/modified-at (Date.)})
 
+(defn completion-meta
+  [user]
+  {:meta/completed-by (user-model/user-ref user)
```

Comment:
Should it be meta attribute or rather a specific attribute that only applies to tasks and task parts?